spyder/plugins/editor/api/folding.py

```diff
@@ -21,10 +21,14 @@
 
 # Standard library imports
 import sys
+import logging
+import re
```

Comment:
Please revert the changes to this file because you didn't really change anything on it.